indigo_app/views/tasks.py

```diff
@@ -62,10 +63,16 @@ def get_context_data(self, **kwargs):
         context['task_groups'] = Task.task_columns(self.form.cleaned_data['state'], context['tasks'])
 
         # potential assignees for tasks. better to batch this here than load it for every task.
-        permitted_users = User.objects.filter(editor__permitted_countries=self.country).all()
+        submit_task_pk = Permission.objects.get(codename='submit_task').pk
+        close_task_pk = Permission.objects.get(codename='close_task').pk
+        potential_assignees = User.objects.filter(editor__permitted_countries=self.country,
```

Comment:
This logic looks fine, the use of primary keys surprises me. Does `User.objects.filter(..., user_permissions=submit_task)` where `submit_task` is the permission itself, and not the pk, not work?

Using objects themselves is generally the preferred Django way.

Reply:
thanks I'll try it! I had tried a couple of other things that didn't work, but not this